src/cloud/project/instance/interface/cloud-project-instance-interface.v6.service.js

```diff
@@ -0,0 +1,38 @@
+angular.module("ovh-api-services").service("OvhApiCloudProjectInstanceInterfaceV6", function ($resource, OvhApiCloudProjectInstanceInterface) {
+
+    "use strict";
+
+    var interceptor = {
+        response: function (response) {
+            OvhApiCloudProjectInstanceInterface.resetCache();
+            return response.data;
+        }
+    };
+
+    var interfacesResource = $resource("/cloud/project/:serviceName/instance/:instanceId/interface/:interfaceId", {
+        serviceName: "@serviceName",
+        instanceId: "@instanceId",
+        interfaceId: "@interfaceId"
+    }, {
+        get: { method: "GET", cache: OvhApiCloudProjectInstanceInterface.cache },
+        query: { method: "GET", cache: OvhApiCloudProjectInstanceInterface.cache, isArray: true },
+        save: { method: "POST", interceptor: interceptor },
+        remove: { method: "DELETE", interceptor: interceptor },
+        "delete": { method: "DELETE", interceptor: interceptor }
```

Comment:
You have two methods for the same purpose ?

Reply:
I think it will be better to use the default set (ref: https://docs.angularjs.org/api/ngResource/service/$resource#$resource-returns). Isn't it?

Reply:
Same here : https://github.com/ovh-ux/ovh-api-services/blob/master/src/cloud/project/snapshot/cloud-project-snapshot.v6.service.js#L36-L37
& here : https://github.com/ovh-ux/ovh-api-services/blob/master/src/cloud/project/instance/cloud-project-instance.v6.service.js#L19-L20

Reply:
We should handle this issue in another PR:

> Search all service where this method is duplicated
> Remove deprecated method (breaking changes)
> Create a major release

Reply:
In that particular case it shouldn't be handled separately because that service cover one specific needs but nevermind. Package was already release.

Of course, all suggestions may help to address and prevent this kind of situation.